README.md

```diff
@@ -1 +1,216 @@
-# Doctor-Reservation-backend
+<a name="readme-top"></a>
```

Comment:
# Frontend review

- [ ]  On the "License" section of your readme, the link is broken because it points to a file that does not exist in your project.

Go [here](https://github.com/microverseinc/readme-template), grab the `MIT.md` file from that repo, paste it on the root of your project, and link to it in the "License" section of your readme.

That way, your app will be well documented 💪

- [ ] Kindly, remove the `console.log` debugging statements found in these files:

![image](https://github.com/Iqbal-Elham/Doctor-Reservation-backend/assets/57421823/60a94223-fd0d-43aa-a58f-2066d13d56d6)

That way, your repo's cleanliness will be improved. 👍 

- [ ] Please, kindly remove the old commented code present in these files:

1. ReservationSlice.js
2. Reserve.js

- [ ] When I try to delete a doctor, the server has an internal error:

![image](https://github.com/Iqbal-Elham/Doctor-Reservation-backend/assets/57421823/d1581140-7096-4cb8-a250-040cf66cb9c2)

Kindly, fix this bug so your app works as intended. 👍 

- [ ] When I add enough doctors, some of them start dissapearing from the screen:

![image](https://github.com/Iqbal-Elham/Doctor-Reservation-backend/assets/57421823/e2c8246c-3d86-4218-b098-c698e35d9abd)

Kindly, implement some for of carrousel, or any other way so the doctors won't disappear (it could also be a horizontal scroll). 

- [ ] In the mobile implement, the hamburger menu does not work, no matter how many times I click on it.  Kindly fix it so your app works as intended. 👍 

- [ ] The mobile styling implementation for the home page and for adding doctors, is questionable:

![image](https://github.com/Iqbal-Elham/Doctor-Reservation-backend/assets/57421823/ab55327f-2a00-4a61-803b-5c0c5c07b300)

![image](https://github.com/Iqbal-Elham/Doctor-Reservation-backend/assets/57421823/415350a0-d521-43aa-9060-43937841e3e4)

Kindly, add  more styling and or touch-ups so they look more professional, and functional.

That way, your mobile view will be beautiful. 💪 

- [ ]  Please, kindly refactor all other functions in your project to ES6 arrow functions, so your code complies with the latest standards (I found 5).

Refactoring is easy:

```javascript
const App = () => {
  // ...
}
```

## Explanation

The `function` keyword has been soft deprecated in almost all the software industry in favor of ES6 arrow functions. This means, almost nobody uses it anymore, so much that it's considered a non-best practice.

You can do a search in Vscode for the "function" word, and correct as needed.  Shouldn't take you more than 5-10 min so don't worry:

![image](https://github.com/Iqbal-Elham/Doctor-Reservation-backend/assets/57421823/0e6ac0ae-c314-440c-b13e-ae03a057ca0e)


That way, your code will meet the latest standards. 💪